src/main/groovy/life/qbic/samplestatus/reporter/services/RealLimsQueryService.groovy

```diff
@@ -75,13 +97,15 @@ class RealLimsQueryService implements LimsQueryService {
     }
   }
   /**
-   * {@InheritDocs}
+   * {@inheritDoc}
+   * @param updatedSince {@inheritDoc}
+   * @return {@inheritDoc}
    */
   @Override
   List<Result<SampleUpdate, Exception>> getUpdatedSamples(Instant updatedSince) {
     SampleSearchCriteria criteria = new SampleSearchCriteria()
     // we make sure that the barcode is set, otherwise the sample is of no interest to us
-    criteria.withProperty("QBIC_BARCODE").thatContains("Q")
+    criteria.withProperty(qbicBarcodeProperty).thatContains("Q")
```

Comment:
```suggestion
    criteria.withProperty(qbicBarcodeProperty).thatStartsWith("Q")
```

Unfortunately, this seems to have gotten lost when we tested. We used it to exclude a sample that was strangely named, but had a "Q" in the middle - although throwing an error might be the correct course of action instead!?